lib/tasks/styles.js

```diff
@@ -39,6 +39,14 @@ module.exports = function(gulp, config) {
 			'include css': true
 		}, config.stylus | {})
 
+		var sassOptions = {}
+
+		if(config.sassNodeModulesAlias) {
+			var sassOptions = assign({}, sassOptions, {
```

Comment:
This redeclaration doesn't make sense. It looks like you are declaring new variable, but you don't. It's already declared

Reply:
Yeah typo